orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/ClusterMatchRule.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.utils;
+
+public class ClusterMatchRule implements Comparable<ClusterMatchRule> {
+  private String account;
+  private String location;
+  private String stack;
+  private String detail;
+  private Integer priority;
+
+  public ClusterMatchRule() {
```

Comment:
I'm a little conflicted here. We can omit it, but then we need to always call the other constructor, which makes the tests a little harder to read because you can't do the nice map constructor bit with groovy, e.g. we'd need to do this:
```groovy
new ClusterMatchRule(account, "*", "*", "*", 1)
```
instead of
```
new ClusterMatchRule(account: account, location: "*", stack: "*", detail: "*", priority: 1)
```

It's probably more consistent with the rest of the codebase, however, to go with the single constructor.